### ISSUE TITLE
Support GHC 8.8.3

### ISSUE THREAD

**Issue by bhipple** [Infinisil/all-hies#55]
This is now the default on NixPkgs, but it looks like we only have 8.8.2.

I wasn't able to find instructions on precisely how to do this in the readme, but I tried doing this:

```diff
diff --git a/nixpkgsForGhc/ghc883 b/nixpkgsForGhc/ghc883
new file mode 100644
index 0000000..b4726a7
--- /dev/null
+++ b/nixpkgsForGhc/ghc883
@@ -0,0 +1 @@
+ca0cb2c43f8231fc8e13ef5f40aad40ddbf4a44b
\ No newline at end of file
```

and then running 

```
  alias update="$(nix-build --no-out-link update.nix)/bin/update"
  update --name stable --revision 1.1
```

but it died with some `stack` errors.

**Comment by infinisil** [comment]
Can't support 8.8.3 without at least HIE itself supporting that version, which it currently doesn't.

**Comment by infinisil** [comment]
See https://github.com/Infinisil/all-hies/issues/52 for the error when trying to run the update script

**Comment by kylegentle** [comment]
Lucky for us, it looks like 8.8.3 support was added today in HIE. :relaxed:

https://github.com/haskell/haskell-ide-engine/pull/1697

**Comment by tofutim** [comment]
so... what's next?

**Comment by bhipple** [comment]
A brave soul has to stare into the abyss of code-generated nix expressions and debug.  It seems we have two paths forward: 

A. Have someone take up maintainership of the now abandoned `stack2nix` generator and fix it, [as done here](https://github.com/Infinisil/all-hies/issues/52#issuecomment-619243792), or

B. Switch to `haskell.nix`, [as discussed here](https://github.com/Infinisil/all-hies/issues/19).

**Comment by Anton-Latukha** [comment]
From exp or measuring the probabilities - it is unreal that anyone would adopt and work on `stack2nix` up to a nice state. It always was half-working and half-going.
https://github.com/input-output-hk/stack2nix/issues/171

It is a big risk to rely on `stack2nix` in technical projects, at this point.

`haskell.nix` advertises Stack support also, is also an IOHK creation, team is large and respectable, development is flowing.

**Comment by infinisil** [comment]
See https://github.com/Infinisil/all-hies/issues/19#issuecomment-631052075 and future comments for updates on haskell.nix builds

**Comment by infinisil** [comment]
With #64 merged, 8.6.5 and 8.8.3 are now the only two supported versions, so I can finally close this :)
